docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,82 @@
+[role="xpack"]
+[[xpack-logs-analysis-page]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+The *Analysis* page in the Logs app allows you to automatically detect some kinds of log anomalies using machine learning.
+
+The analysis automatically highlights periods where the log rate is outside the expected limits and therefore may be anomalous.
+This helps you to spot suspicious behavior without significant human intervention.
+You can use this information as a basis for further investigations.
```

Comment:
This could be various things:

- A spike in the log rate could denote a DDoS attack. This may lead to investigating things like IP addresses from incoming requests. 
- A significant drop in the log rate could suggest that some some piece of infrastructure has stopped responding, and thus we're serving less requests. 

These are just examples, mileage will vary between datasets and anomalies.